docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -202,42 +203,32 @@ def my_flow():
 
 To support this, Prefect 3 has more eager, intelligent task caching defaults than Prefect 2.
 Whenever a task runs, Prefect 3 first computes its cache key, then uses that cache key to evaluate whether that same task has run with the same inputs and as part of the same flow before. 
-If it has, the cached result from the previous run is used. If it hasn't, the task runs. 
+If it has, the cached result from the previous run is used. If it hasn't, the task runs.
 
-This means that tasks that may have re-run with the same inputs in Prefect 2 do not re-run in Prefect 3 unless the cache key is modified to be more precise.
+Tasks that may have re-run with the same inputs in Prefect 2 do not re-run in Prefect 3 unless the cache key is modified to be more precise.
 The cache key can be configured through the new global `cache_policy` setting, or through the `cache_policy` on a particular task.
-If a particular task cannot be made idempotent, you can disable this behavior by setting `cache_policy=NONE` on the task.
-
-Results and cache keys are not stored with your Prefect Cloud account or Prefect server instance. 
-They are persisted entirely in your configured result location. 
+If a particular task isn't idempotent, you can disable this behavior by setting `cache_policy=NONE` on the task.
 
 This caching behavior only takes effect when result persistence is enabled.
-The simplest way to enable result persistence is through the Prefect setting `PREFECT_RESULTS_PERSIST_BY_DEFAULT=true`.
-
-See the [caching documentation](/3.0rc/develop/write-tasks#caching) for more information.
-
-## Data validation with Pydantic V2
-
-Prefect 2 uses Pydantic V1 to validate data, particularly for blocks and work pools.
-
-Prefect 3 uses Pydantic V2. This upgrade does not require changes for any built-in Prefect interfaces or supported integrations. 
-However, if you have registered custom blocks with a self-hosted Prefect Server, type enforcement for those block schemas must be upgraded to Pydantic V2 to work with Prefect 3. 
-
-See the [Pydantic migration guide](https://docs.pydantic.dev/latest/migration/) for more information.
+See the [caching documentation](/3.0rc/develop/task-caching) for more information.
 
 ## Workers replace agents
 
 A deprecation warning was applied to Prefect interfaces related to agent-based deployments in Prefect’s [2.16.4 release](https://github.com/PrefectHQ/prefect/releases/tag/2.16.4), including infrastructure blocks and `prefect deployment` commands. 
-Agents continue to be supported in Prefect 2 until September 2024, but are not included in Prefect 3. 
-
+Agents will continue to be supported in Prefect 2 until September 2024, but are not included in Prefect 3. 
 If you are using agents, learn how to [upgrade from agents to workers](/3.0rc/resources/upgrade-agents-to-workers).
 
-## Variables can be any JSON-compatible type
+## Type enforcement via Pydantic 
```

Comment:
@serinamarie I updated the language to reflect the broader use of Pydantic. Feedback welcome

Reply:
yes!!! great